ambari-server/src/main/resources/common-services/AMBARI_INFRA_SOLR/0.1.0/configuration/infra-solr-env.xml

```diff
@@ -182,6 +182,13 @@
     </value-attributes>
     <on-ambari-upgrade add="false"/>
   </property>
+  <property>
+    <name>infra_solr_java_stack_size</name>
+    <value>1024k</value>
```

Comment:
Shouldn't this property have type/unit/min/max/increment attributes like `infra_solr_minmem` and `infra_solr_maxmem` do?

Reply:
it is not in smartconfig right now, so we cannot use those values effectively

Reply:
What is smartconfig?

Reply:
where you can see the nice widgets on the config page. the memory values are used like that